packages/core/src/sequence.ts

```diff
@@ -16,14 +16,51 @@ import {
 import {parseOperationArgs} from './parser';
 import {writeResultToResponse} from './writer';
 import {HttpError} from 'http-errors';
-
+/**
+ * The default implementation of Sequence.
+ *
+ * This class implements default Sequence for the LoopBack framework.
+ * Default sequence is used if user hasn't defined their own Sequence
+ * for their application.
+ *
+ * Sequence constructor() and run() methods are invoked from [[http-handler]]
+ * when the API request comes in. User defines APIs in their Application
+ * Controller class.
+ *
+ * Sequence executes these steps
```

Comment:
regarding my comment below, maybe we can just mention this set of steps from the `run()` method